packages/flutter/lib/src/material/input_date_picker_form_field.dart

```diff
@@ -239,9 +239,13 @@ class _InputDatePickerFormFieldState extends State<InputDatePickerFormField> {
   Widget build(BuildContext context) {
     final MaterialLocalizations localizations = MaterialLocalizations.of(context);
     final InputDecorationTheme inputTheme = Theme.of(context).inputDecorationTheme;
```

Comment:
You could separate out a `theme` final var here too.

Reply:
Addressed in https://github.com/flutter/flutter/pull/119033/commits/70d860d720f506a57c730749eaaba252e4cd8a6b